internal/app/transport/request.go

```diff
@@ -107,9 +111,14 @@ func (r *RequestV2) GetSubZone() string {
 	return r.r.GetNode().GetLocality().GetSubZone()
 }
 
-// GetRaw gets the error details
-func (r *RequestV2) GetRaw() *RequestVersion {
-	return &RequestVersion{V2: r.r}
+// GetVersionedRequest gets the error details
+func (r *RequestV2) getVersionedRequest() *requestVersion {
+	return &requestVersion{V2: r.r}
+}
+
+// GetRequest gets the error details
+func (r *RequestV2) GetRequest() interface{} {
```

Comment:
This is the one place we had to use `interface{}` because of its use in orchestrator and http handler.